test/plain-list/AutosuggestApp.test.js

```diff
@@ -140,13 +237,84 @@ describe('Plain list Autosuggest', () => {
       expectSuggestions([]);
     });
 
+    it('should not focus suggestions', () => {
+      expectFocusedSuggestion(null);
+    });
+
+    it('should clear the input when Escape is pressed', () => {
+      clickEscape();
+      expectInputValue('');
+    });
+  });
+
+  describe('when typing and matches do not exist (when selectFirstSuggestion is true)', () => {
+    before(() => {
+      setSelectFirstSuggestion(true);
+    });
+
+    after(() => {
+      setSelectFirstSuggestion(false);
+    });
+
+    beforeEach(() => {
+      focusAndSetInputValue('z');
+    });
+
+    it('should not show suggestions', () => {
+      expectSuggestions([]);
+    });
+
+    it('should not focus suggestions', () => {
+      expectFocusedSuggestion(null);
```

Comment:
Admittedly, right now `expectFocusedSuggestion` checks for the existence of DOM elements but that may be updated to include checking the focused index in the component to be more thorough.